Solutions/FlareSystemsFirework/Playbooks/azuredeploy.json

```diff
@@ -0,0 +1,283 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "credential-warning", 
+        "author": {
+            "name": "Jean-Christophe Taillandier"
+        }
+    },
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "Credential Warning",
```

Comment:
Name should not have spaces in between. You ca use '-' | '_' | '.' 
Please update it accordingly.

Reply:
Sorry about that!